kuma/static/js/wiki.js

```diff
@@ -679,4 +679,47 @@
         }
     }
 
+    // listens for post message from interactive editor
+    window.addEventListener('message', function(event) {
+        // get the interactive editor iframe
+        var iframe = document.querySelector('iframe.interactive');
+        var isExpectedOrigin = event.origin === 'https://interactive-examples.mdn.mozilla.net';
+
+        /* there may be other post messages so, ensure that the origin is the
+        expected and, that `event.data` contains an `iframeHeight` property */
+        if (isExpectedOrigin && event.data.iframeHeight) {
+            iframe.setAttribute('height', event.data.iframeHeight);
+        }
+    }, false);
+
+    // Insert notice about permanent language switch
+    var changedLocaleTo = localStorage.getItem('changed-locale-to');
```

Comment:
This code also needs to appear in ``wiki-skinny.js``, or be moved to a new file that is included in both ``wiki-skinny.js`` and ``wiki.js``.

Reply:
+1 Moving this code to a separate file. It should be part of lang-switcher.js